src/marketing/recommendation-incl-excl.md

```diff
@@ -0,0 +1,43 @@
+
+{:.bs-callout-info}
+Enabling inclusion and exclusion conditions requires that you have version 3.11.0 or later of the `magento/product-recommendations` module. See the [release notes](https://devdocs.magento.com/recommendations/release-notes.html) to learn more.
+
+## Types of conditions
+
+Magento provides the following inclusion and exclusion conditions you can use to control which products are displayed in recommendations.
+
+- **Category** - Filters based on a product's category. Magento uses direct category assignments and their subcategories. For example, if the category of the product is `Gear`, enabling this rule includes or excludes products that have categories under `Gear`, such as `Gear/Bags` or `Gear/Fitness Equipment`.
+- **Price** - Filters based on a product's price. Magento uses a product's final price when performing the comparison. The final price includes any discounts or special pricing applied.
+- **Product** - Specifies which products are eligible or not eligible to be displayed.
+- **In stock status/Low stock** - (Available as an exclusion only.) Filters based on the **Display Out-of-Stock Products** value configured in the Magento Admin to determine if a particular product should be displayed in a recommendation unit. If the **Display Out-of-Stock Products** option is set to `Yes`, products that are out-of-stock are eligible to be displayed in the recommendation unit. If **Display Out-of-Stock Products** option is set to `No`, out-of-stock products will not be displayed in recommendation units. You can configure the **Display Out-of-Stock Products** value in the [stock options]({% link configuration/catalog/inventory.md %}) section of the Catalog configuration.
+- **Type** - Filters products based on type, such as: **Simple**, **Configurable**, **Virtual**, **Downloadable**, or **Gift card**. **Bundled** and **Grouped** products are not yet supported.
+- **Visibility** - Filters products based on visibility, such as: **Catalog**, **Search**, or both.
+
+## Setting multiple conditions
+
+If using both inclusion and exclusion conditions, Magento applies AND/OR logic operators as per the following:
+
+- ANDs for inclusions
+- ORs for exclusions
+- ORs with multi-select within inclusions or within exclusions
+
+For example, let's say you want to Provide basic examples of the more common multiple filter combinations.
+what if user creates identical exclusion to an inclusion? ui/magento allows it...need to explain what happens.
+
+## Static versus dynamic conditions {#staticdyn}
```

Comment:
not sure how I feel about this table. it's pretty confusing in its current form. maybe we take it out and just say dynamic ones are available for recs on every page type except home and Pagebuilder?